api/v1/db/oracledb/LearningDB.sql

```diff
@@ -0,0 +1,192 @@
+DROP TABLE CARD_SUITS;
+DROP TABLE CARD_NUMBERS;
+DROP TABLE GAMES;
+DROP TABLE MEMBERS;
+DROP TABLE PARTICIPATIONS;
+DROP TABLE ROUNDS;
+DROP TABLE STATUSES;
+
+
+CREATE TABLE CARD_SUITS (
+    SUIT_ID NUMBER NOT NULL,
+    SUIT    VARCHAR2(255 BYTE),
+    PRIMARY KEY (SUIT_ID)
+);
+
+COMMENT ON TABLE CARD_SUITS IS 'Contains possible values for card suit';
+COMMENT ON COLUMN CARD_SUITS.SUIT_ID IS 'ID of suit';
+COMMENT ON COLUMN CARD_SUITS.SUIT IS 'Suit name';
+
+INSERT INTO CARD_SUITS (SUIT_ID, SUIT) VALUES (1,'spades');
+INSERT INTO CARD_SUITS (SUIT_ID, SUIT) VALUES (2,'diamonds');
+INSERT INTO CARD_SUITS (SUIT_ID, SUIT) VALUES (3,'clubs');
+INSERT INTO CARD_SUITS (SUIT_ID, SUIT) VALUES (4,'hearts');
```

Comment:
@tsoliangwu0130 mentioned that Banner tables will use IDs like `'E'` for employed and `'T'` for terminated and have another column with a description field. Using something like this would be more meaningful than `1`, `2`, `3`, and `4`